scripts/deposit.js

```diff
@@ -24,12 +25,13 @@ module.exports = async (callback) => {
     }
   
     const token = await ERC20Mintable.at(token_address)
-    const depositor_balance = (await token.balanceOf.call(depositor)).toNumber()
-    if (depositor_balance < amount) {
-      callback("Error: Depositor has insufficient balance.")
+    const depositor_balance = (await token.balanceOf.call(depositor))
+    if (depositor_balance.lt(new web3.utils.BN(amount))) {
+      console.log(`Deposit request of ${amount} < `)
```

Comment:
Do you want to keep this?

Reply:
Sorry, I forgot to include some changes (vsCode doesn't save auto-magically).